public/chore.js

```diff
@@ -0,0 +1,32 @@
+function clearFields() {
+    const elements = document.getElementById('chore-form').elements;
+
+    for (i = 0; i < elements.length; i++) {
+        fieldType = elements[i].type.toLowerCase();
+        switch (fieldType) {
+        case 'text':
+            elements[i].value = '';
+            break;
+        case 'checkbox':
+            if (elements[i].checked) {
+                elements[i].checked = false;
+            }
+            break
+        default:
+            break;
+        }
+    }
+}
+
+
+const choreList = ["Make Bed", "Walk Dogs", "Clean Room", "Empty Dishwasher"];
```

Comment:
I'd like this to be an array of objects.

`const choreList = [{ name: "Make Bed", assignee:  "Kyle"} ];`  // add 2 more